res/themes/light-custom/css/_custom.pcss

```diff
@@ -26,16 +26,10 @@ $secondary-content: var(--secondary-content, $secondary-content);
 $tertiary-content: var(--tertiary-content, $tertiary-content);
 $quaternary-content: var(--quaternary-content, $quaternary-content);
 $quinary-content: var(--quinary-content, $quinary-content);
-/* XXX: workaround for Safari 15.3 linear-gradient bug https://github.com/vector-im/element-web/issues/21670 */
-$system-transparent: var(--system-transparent, rgba($system, 0));
 $system: var(--system, $system);
 $background: var(--background, $background);
-$panels: rgba($system, 0.9);
-$panel-base: var(--panel-base, $tertiary-content); /* This color is not intended for use in the app */
-$panel-selected: rgba($panel-base, 0.3);
-$panel-hover: rgba($panel-base, 0.1);
-$panel-actions: rgba($panel-base, 0.2);
-$space-nav: rgba($panel-base, 0.1);
+$panels: var(--cpd-color-gray-600);
+$panel-actions: var(--cpd-color-gray-300);
```

Comment:
I don't think we should be leaking `--cpd-X` into custom theme var naming, this could be the default value of course but I wouldn't expect to need to use `"cpd-color-gray-600": foo` in the JSON.

Reply:
Makes sense! 

changed to have `--panels` and use `--cpd-color-gray-X` as the default value